### PR TITLE
Enhancement better logging2

### PR DESCRIPTION
**Pull request by DSWallach**
# Pull Request Template for MMEDS

## What has changed
Include each file that was changed, how it was changed (in broad terms), and why it was changed.
Changed log.py to logging.py. Now uses setup similar to ensemble

## Checklist of pre-requisites
-   [x] Does the code run?  
-   [x] Does the code follow the repository style?  
-   [x] Is the code tested?  

## How to use the feature
Example: To do X run new function Y with parameters A, B, and C which gives result Z.

## How the design has changed
Example: Functions A, B, and C are now all methods of Class D.

### CODE REVIEW

File: mmeds/resources/log_config.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+handlers:
+    console:
+        class: logging.StreamHandler
+        formatter: default
+        level: DEBUG
+    file:
+        class: logging.handlers.RotatingFileHandler
+        level: DEBUG
+        formatter: default
+        filename: /home/david/Work/ensemble/logging/ensemble_debug.log
```
Comment (mstapylton) [inline mmeds/resources/log_config.yaml:14]: I'm pretty sure this is being set somewhere, and could therefore be blank. Might be better that way to key people off that `filename` is being set programmatically.

File: mmeds/tests/server/test_server.py [hunk 2]
```diff
@@ -625,36 +623,33 @@ def check_analysis_has_privileges(self):
...
+        self.getPage("/study/select_study", headers=self.cookies)
+        # Parse the body of the page
+        body = self.body.decode('utf-8')
+        # Grab the access code
+        code = re.findall('access_code=(.+?)">', body)
+        # Check that it works to access the view_study page
+        self.getPage("/study/view_study?access_code={}".format(code[1]), headers=self.cookies)
         self.assertStatus('200 OK')
-        self.getPage('/logout', headers=self.cookies)
+        breakpoint()
```
Comment (mstapylton) [inline mmeds/tests/server/test_server.py:649]: breakpoint be gone!

**Review by mstapylton — CHANGES_REQUESTED** [review]
couple small things, otherwise looks good

File: mmeds/resources/log_config.yaml (hunk 1 above)
Comment (DSWallach) [reply mmeds/resources/log_config.yaml:14]: tis

**Review by mstapylton — APPROVED** [review]

**Pull request merged**